### DISCUSSION TITLE
画面遷移図

### DISCUSSION THREAD

**Discussion by topi-log** [UNNNYOU/Project-RQB#2] [アイデア]
「回答」ページのPC、SP版の画面を作ってみました。
色合い・方向性の確認の意味も込めてDiscussionに投稿してみました。

https://www.figma.com/design/keyhPg4SkwguvphZOPtPNx/%E3%80%80RUNTECQ-overfofw?node-id=0-1&t=QuFjjx95YZgQVk04-1

画像版
左がPC、右がSPです。
![image](https://github.com/UNNNYOU/Project-RQB/assets/23026318/bef35075-c607-4400-ab38-8f5929803079)

補足
細かい部分のコメントは上記figmaのほうだと助かります。

以上です。よろしくお願いします。

**Comment by unnnyou** [comment]
しごでき！！さすが！！
actionの設定が一段落したら確認させていただきます！
